scripts/run-local-test

```diff
@@ -0,0 +1,103 @@
+#!/usr/bin/env bash
+# Copyright 2019 Red Hat, Inc.
+# All Rights Reserved.
+#
+# Licensed under the Apache License, Version 2.0 (the "License"); you may
+# not use this file except in compliance with the License. You may obtain
+# a copy of the License at
+#
+#     http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS, WITHOUT
+# WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied. See the
+# License for the specific language governing permissions and limitations
+# under the License.
+
+
+## Shell Opts ----------------------------------------------------------------
+
+set -o pipefail
+set -xeuo
+
+## Vars ----------------------------------------------------------------------
+
+export PROJECT_DIR="$(dirname $(readlink -f ${BASH_SOURCE[0]}))/../"
+export ROLE_NAME="${ROLE_NAME:-$1}"
+export TRIPLEO_JOB_ANSIBLE_ARGS=${TRIPLEO_JOB_ANSIBLE_ARGS:-""}
+export TOX_CONSTRAINTS_FILE=${TOX_CONSTRAINTS_FILE:-"https://releases.openstack.org/constraints/upper/master"}
+
+# NOTE(cloudnull): Disable ansible compat check, caters to the case where
+#                  system ansible may be installed.
+export ANSIBLE_SKIP_CONFLICT_CHECK=1
+
+## Functions -----------------------------------------------------------------
+
+function run_pip {
+    "${HOME}/test-python/bin/pip" install \
+                                  -c "${TOX_CONSTRAINTS_FILE}" \
+                                  -c "${PROJECT_DIR}/ansible-requirements.txt" \
+                                  -r "${PROJECT_DIR}/requirements.txt" \
+                                  -r "${PROJECT_DIR}/test-requirements.txt" \
+                                  -r "${PROJECT_DIR}/molecule-requirements.txt" ${@:-}
+    ansible-galaxy collection install ansible.posix \
+                                  community.general \
+                                  containers.podman                                  
```

Comment:
Replace this with `ansible-galaxy collection install -r ${PROJECT_DIR}/requirements.yml`, otherwise you should maintain a list of requirements in two different places.

Reply:
I've replaced it now. I was also calling the command out of the venv. Thank you.